libvast/src/format/zeek.cpp

```diff
@@ -475,6 +479,9 @@ caf::error reader::parse_header() {
   }
   // Add #index=hash attribute for fields where it makes sense.
   add_hash_index_attribute(layout_);
+  for (auto i = 0u; i < layout_.fields.size(); ++i)
+    VAST_DEBUG(this, "     ", i << ')', layout_.fields[i].name << ':',
+               layout_.fields[i].type);
```

Comment:
Do the spaces in the debug message have a purpose?

Reply:
I only moved that part down a couple of lines.